### PR TITLE
Add prometheus-ai-project for team promethues-ai

### PR DESCRIPTION
**Pull request by benlugasi**
Hey,
I've added a project for our group (that was added earlier) following [this](https://github.com/operate-first/hitchhikers-guide/blob/main/pages/onboarding_project.ipynb) guide.

### CODE REVIEW

File: cluster-scope/base/core/namespaces/prometheus-ai-project/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+    - namespace.yaml
+components:
+    - ../../../../components/project-admin-rolebindings/prometheus-ai
+    - ../../../../components/limitranges/default
```
Comment (harshad16) [inline cluster-scope/base/core/namespaces/prometheus-ai-project/kustomization.yaml:7]: please also add quota: small, medium, large

**Review by harshad16 — CHANGES_REQUESTED** [review]
lgtm
some changes needed:

File: cluster-scope/base/core/namespaces/prometheus-ai-project/kustomization.yaml (hunk 1 above)
Comment (benlugasi) [reply cluster-scope/base/core/namespaces/prometheus-ai-project/kustomization.yaml:7]: This was not mentioned in our guides, can you please specify what do you mean?
Do you want us to add key 'quota' value 'medium' for this yaml file? or maybe something else.
If there is any example that would be great

**Comment by benlugasi** [comment]
Changes applied @harshad16

**Review by harshad16 — COMMENTED** [review]
lgtm :100:

**Comment by benlugasi** [comment]
> @benlugasi I will work on updating the docs for quotas This PR looks good 👍 Can you just squash the commits to one?

Sure, Thanks

**Comment by benlugasi** [comment]
squashed

**Review by harshad16 — COMMENTED** [review]
/lgtm

**Review by 4n4nd — APPROVED** [review]
/lgtm

**Pull request merged**